swagger.yaml

```diff
@@ -1,75 +1,208 @@
-swagger: '2.0'
+swagger: "2.0"
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
+  description: "This API allows users to view information on textbooks associated with course sections."
+  version: "1.0"
+  title: "Textbooks API"
+
+tags:
+- name: "textbooks"
+  description: "Contains information on textbooks"
+
 schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
+- "https"
+
+host: "api.oregonstate.edu"
+
+basePath: "/v1"
+
+externalDocs:
+  description: "GitHub Repository"
+  url: "https://github.com/osu-mist/textbooks-api"
+
 produces:
-  - text/plain
+  - "application/json"
+
 paths:
-  /:
+
+  /textbooks/{id}:
     get:
-      summary: Get information
-      description: Build and runtime information about the application.
+      tags:
+      - "textbooks"
+      summary: "Retrieve textbooks by ID"
+      operationId: "getTextbooksById"
       parameters:
-        - $ref: '#/parameters/pretty'
+        - name: "id"
+          in: "path"
+          description: "Textbook ID"
+          required: true
+          type: "string"
+
       responses:
         200:
-          description: Object containing information
+          description: "Textbook successfully retrieved"
+          schema:
+            $ref: "#/definitions/TextbookResult"
+        404:
+          description: "Textbook not found"
           schema:
-            $ref: '#/definitions/Info'
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
+
+  /textbooks:
+    get:
+      tags:
+      - "textbooks"
+      summary: "Retrieve textbooks by parameters"
+      operationId: "getTextbooks"
+      parameters:
+
+        - name: "term"
+          in: "query"
+          description: "Name of term in which to find textbooks"
+          type: "string"
+          required: true
+
+        - name: "department"
+          in: "query"
+          description: "Name of department in which to find textbooks"
+          type: "string"
+          required: true
 
-parameters:
-  pretty:
-    name: pretty
-    in: query
-    type: boolean
-    required: false
-    description: If true, JSON response will be pretty-printed
+        - name: "course"
```

Comment:
Similar comment here, could you change this parameter name to match our class search API? https://github.com/osu-mist/class-search-api/blob/ccb438057876109d01f79462371537f2b4eafbb9/swagger.yaml#L23-L32